examples/advanced_operations/add_dynamic_search_ads.py

```diff
@@ -207,12 +204,11 @@ def create_expanded_dsa(client, customer_id, ad_group_resource_name):
                                                     [ad_group_ad_operation])
```

Comment:
should probably unindent a bunch of spaces. The current indentation is inconsistent